packages/jsonp/src/index.ts

```diff
@@ -25,8 +25,6 @@ function stringify(
           return '\\u003e'
         case 0x26:
           return '\\u0026'
-        default:
```

Comment:
why remove default case? it should fallback if switch falls through

Reply:
The regex `/[<>&]/g` won't allow any other char. Hence, it won't reach the default.